pkg/compliance/checks/command_utils_unix.go

```diff
@@ -0,0 +1,82 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !windows
```

Comment:
I think this is a >1.17 build constraint

Reply:
I think editors, or some go tool, are adding this automatically?  I see it added in lots of PRs.

Reply:
I believe it's `gofmt` from go 1.17. It's harmless and we'll need to fix all constraints when migrating to 1.17 so this doesn't seem blocking to me